### PR TITLE
[PT-BR] Sharable URLs.ts, TypeScript Versions.ts

### PR DESCRIPTION
**Pull request by luan-sabino**
@khaosdoctor
@alvarocamillont
@danilofuchs
@orta

### CODE REVIEW

File: packages/playground-examples/copy/pt/Playground/Tooling/Sharable URLs.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+// Com o novo Playground, os URLs tem muito mais informações
+// sobre suas configurações. Isso faz com que reproduções
+// sejam precisas, e que você não tenha que falar para alguém
+// usar uma determinada flag de compilador para replicar a
+// sua configuração.
+
+// A URL contem:
+//
+// - Se você esta em um exemplo
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Tooling/Sharable URLs.ts]:
```suggestion
// - Se você está em um exemplo
```

File: packages/playground-examples/copy/pt/Playground/Tooling/Sharable URLs.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+// - O código-fonte compactado para o seu TypeScript/JavaScript
+// - Retrocompatilibilidade com URLs antigas não compactadas
+// - Em qual idioma você está trabalhando
+// - A diferença entre suas configurações de compilação e 
+//   a configuração padrão
+//   Por exemplo:new-compiler-defaults
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Tooling/Sharable URLs.ts]:
```suggestion
//   Por exemplo: example:new-compiler-defaults
```
`example:*` are links and must not be translated directly

File: packages/playground-examples/copy/pt/Playground/Tooling/Sharable URLs.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+// O URL é automaticamente atualizado quando ocorre alguma mudança
+// nos tópicos acima, utilizando o replaceState do HTML5, então
+// o seu botão de voltar vai continuar funcionando como esperado.
+
+// Quando um exemplo é carregado, você ocasionalmente verá e=[numero]
+// que é usado para garantir que o navegador recarregue corretamente
+// quando você alternar entre os exemplos. É inofensivo.
+
+// Você pode ver a mudança do URL nessa pagina editando alguns dos
+// comentários, ou selecionando algum texto. Ao recarregar a pagina
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Tooling/Sharable URLs.ts]:
```suggestion
// comentários, ou selecionando algum texto. Ao recarregar a página
```

File: packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+// Com o novo Playground, nos temos muito mais controle sobre
+// o ambiente em que seu código é executado. O novo Playground
+// esta agora vagamente acoplado ao monaco-editor e ao
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts]:
```suggestion
// está agora levemente acoplado ao monaco-editor e ao 
```

File: packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
+// Com o novo Playground, nos temos muito mais controle sobre
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts]:
```suggestion
// Com o novo Playground, nós temos muito mais controle sobre
```

File: packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+// o ambiente em que seu código é executado. O novo Playground
+// esta agora vagamente acoplado ao monaco-editor e ao 
+// monaco-typescript que fornecem a experiencia de edição
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts]:
```suggestion
// monaco-typescript que fornecem a experiência de edição
```

File: packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+// https://github.com/microsoft/monaco-editor/
+// https://github.com/microsoft/monaco-typescript
+
+// Vagamente acoplado significa que o playground permite que o
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts]:
```suggestion
// Levemente acoplado significa que o playground permite que o
```

File: packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+// usuário tenha suporte ao uso de diferentes versões
+// do TypeScript que o monaco-typescript tem integrado.
+
+// Nós temos a infraestrutura para criar uma copia do monaco-editor
```
Comment (danilofuchs) [inline packages/playground-examples/copy/pt/Playground/Tooling/TypeScript Versions.ts]:
```suggestion
// Nós temos a infraestrutura para criar uma cópia do monaco-editor
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]
Nice!
Just some minor tweaks

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**